tests/logictest/README.md

```diff
@@ -0,0 +1,26 @@
+
+# Targets
+
+The database return right with diffrent handlers, for example mysql and http
+
+# Usage
+
+## prepare
+1. Change to the scripts dir, cd tests/logictest/
+2. Make sure python3 is installed
+3. Using pip to install dependency, pip install -r ./requirements.txt  (use -i for private repo)
+
+## Need to Known
+1. Cases from **tests/suites/0_stateless/**  to  **tests/logictest/suites/gen/**
```

Comment:
If we already gen cases from stateless dir, should we keep the old stateless tests ?

Reply:
Unless the logictest framework is perfect, before that, the two mechanisms are parallel

Reply:
Unless the logictest framework is perfect, before that, the two mechanisms are parallel. Better keep those.

Reply:
Ok. Almost LGTM.  

If this pr is merged, which test style should we use?

Reply:
sqllogictest is better, developers can see every single case results or error(code or msg). But it's a long term to make it good enough.

Reply:
Ok, since two version tests will make developers confuse and poor maintenance. To keep fewer duplicated tests I suggest:

1. Delete the `stateless` tests directory if this pr is merged.
2. Or we generate `gen` directories on the fly and keep this directory git ignore.

Reply:
We should do it, or the stateless will add new tests and need to sync the test files with logictest. @ZeaLoVe